components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/KeyManagersApiServiceImpl.java

```diff
@@ -230,6 +232,29 @@ public Response keyManagersPost(KeyManagerDTO body, MessageContext messageContex
         return null;
     }
 
+    private void validateIdpTypeFromTokenType(KeyManagerConfigurationDTO keyManagerConfigurationDTO)
+            throws APIManagementException {
+        String tokenType = keyManagerConfigurationDTO.getTokenType();
+        String keyManagerType = keyManagerConfigurationDTO.getType();
+        if (StringUtils.equals(tokenType.toLowerCase(),
+                KeyManagerConfiguration.TokenType.EXCHANGED.toString().toLowerCase())) {
```

Comment:
shall we simply use string equalignore case

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/10612/commits/4c7facf798f8e19a3aca817a3f3a07ab5623c05a